medusa/subtitles.py

```diff
@@ -439,12 +439,6 @@ def download_subtitles(tv_episode, video_path=None, subtitles=True, embedded_sub
     except IOError as error:
         if 'No space left on device' in ex(error):
             logger.warning(u'Not enough space on the drive to save subtitles')
-        else:
-            logger.warning(traceback.format_exc())
-    except Exception as error:
-        logger.debug(u'Exception: %s', error)
-        logger.info(u'Error occurred when downloading subtitles for: %s', video_path)
-        logger.error(traceback.format_exc())
```

Comment:
You should remove it. The current way you're just swalling the IOError exception without logging it.